python-sdk/tests_integration/test_example_dags.py

```diff
@@ -88,8 +88,9 @@ def order(dag_id: str) -> int:
 
 @pytest.mark.parametrize("dag_id", sorted(dag_bag.dag_ids, key=order))
 def test_example_dag(session, dag_id: str):
-    dag = dag_bag.get_dag(dag_id)
-    wrapper_run_dag(dag)
+    if dag_id == "example_load_file":
+        dag = dag_bag.get_dag(dag_id)
+        wrapper_run_dag(dag)
```

Comment:
I assume this is only temporarily for local testing. But actually you could have just used `test_example_dag[example_load_file]`.